02-express-tutorial/public/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Product List</title>
+</head>
+<body>
+    <h1> Welcome to My simple page</h1>
+    <button id="fetchProductsBtn">Fetch Products</button>
```

Comment:
This button does nothing from what I see, I know it mentions trying some logic, but maybe you didn't get around to it? Not sure but please only submit code that is to be reviewed. Otherwise, leave a note or something that indicates your work in progress. When it comes to code reviews, you are not the only one you are writing code for.